util/microservice-sdk/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dojot/microservice-sdk",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
We will bump the version when we merge to the the master and publish the package.